src/components/common/animations/ClaimButton/ClaimButton.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import Lottie from 'lottie-react-native'
+import { Platform, TouchableOpacity } from 'react-native'
+import _set from 'lodash/set'
```

Comment:
`import { lodash, functions, you, need } from 'lodash';`

you shouldn't import functions one-by-one due ti the performance reasons as there is an webpack/babel plugins that optimizes your imports

just install them and you don't need to perform manual optimizations
https://github.com/lodash/lodash-webpack-plugin
https://github.com/lodash/babel-plugin-lodash